packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -1,29 +1,53 @@
-import { Button, Box, Grid } from 'grommet'
-import { FormNext, FormPrevious } from 'grommet-icons'
+import { Button, Box, Grid, Select, ThemeContext } from 'grommet'
+import {
+  CirclePlay,
+  Pause,
+  FormNext,
+  FormPrevious,
+  FormDown
+} from 'grommet-icons'
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import styled, { withTheme } from 'styled-components'
 import { useTranslation } from 'react-i18next'
 
+import controlsTheme from './theme'
 import locationValidator from '../../../helpers/locationValidator'
 
-const StyledButton = styled(Button)`
+const DirectionButton = styled(Button)`
```

Comment:
```suggestion
const DirectionButton = styled(Button)`
  border: ‘none’;
  padding: 0;
```

You can add your border and padding here, or use Grommet’s `border` and `pad` props on the buttons.